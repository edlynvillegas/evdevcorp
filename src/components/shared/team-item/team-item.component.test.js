import React from 'react'
import renderer from 'react-test-renderer';
import TeamItem from './team-item.component'

test('TeamItem renders correctly', () => {
    const tree = renderer
      .create(<TeamItem/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});