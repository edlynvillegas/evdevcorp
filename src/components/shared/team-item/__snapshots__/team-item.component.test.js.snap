// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamItem renders correctly 1`] = `
<div
  className="team-item grid__col-4 grid__col-mb-12"
>
  <img />
  <div
    className="item-info"
  >
    <div>
      <figure>
        <img />
      </figure>
      <p />
      <p />
      <p />
    </div>
    <div />
  </div>
</div>
`;
